locales/en.yml

```diff
@@ -50,6 +50,9 @@ en:
           on_call_lookup:
             syntax: pager oncall <schedule>
             desc: Show who is on call for the given schedule
+          base_lookup:
+            syntax: pager base <schedule>
+            desc: Show who the base on-call person is for the given schedule
```

Comment:
nit: we have a mix of `on-call` and `on call` (below)

Reply:
Good catch.

Reply:
There were significantly more references to "on call" than to "on-call", so I've moved to that.